mozilla_vpn/views/subscriptions.view.lkml

```diff
@@ -153,7 +170,7 @@ view: +subscriptions {
 
   dimension: retention {
     hidden: yes
-    sql: GENERATE_ARRAY(0, ${months_retained});;
+    sql: GENERATE_ARRAY(0, ${current_age_in_months});;
```

Comment:
new retention dashboards need to be able to count subscriptions up to current age, not just up to `months_retained`